### DISCUSSION TITLE
The turbo.json config does not reflect changes

### DISCUSSION THREAD

**Discussion by douglasg14b** [vercel/turborepo#8467] [Help]
### Summary

When I change `turbo.json` those changes are not reflected on the next run of turbo.

It seems to have cached it from the first run, and every subsequent run uses the cached config when its triggered via `npx` such as `npx lint-staged`

How do I stop this behavior?

Turbo `2.0.3`

![image](https://github.com/vercel/turbo/assets/1400380/302c1525-098e-4871-83f7-c6751dc290d6)

**Comment by versecafe** [comment]
@douglasg14b Not sure where you're deploying but turbo repo sometimes doesn't invalidate caches for changes to JSON files there's an issue about this breaking Nextra somewhere on github but you just need to run with `--force` to ignore cache in the build which should resolve this.

**Reply by anthonyshew** [reply]
The issue you're describing is unique to Nextra. It doesn't have anything to do with Turborepo.

**Comment by anthonyshew** [comment]
Turborepo partitions the cache on changes to `turbo.json`. As long as `turbo.json` is different than a previous run, tasks will miss cache.

The screenshot on the right is invoking Turborepo at a 1.x version. I'm not sure how you have your repo configured such that that would be happening, but you'd want to look for a reason that 1.x would still be getting used in your repository.
